less/pages/safety/safety.less

```diff
@@ -105,6 +105,26 @@
       margin: 0 0.2rem 0 0;
       height: 1rem;
     }
+    input[type=checkbox] {
+      display: none;
+    }
+    label:before {
+      content: "";
+      display: inline-block;
+      width: 24px;
+      height: 24px;
```

Comment:
I dunno, I set this to 1.1rem locally, and seemed to work and is a pretty nice simple rem.